### PR TITLE
add uniprot endpoints

### PR DESCRIPTION
**Pull request by leexgh**
Fix https://github.com/genome-nexus/genome-nexus/issues/441
For now only fetch data on the fly.

image

### CODE REVIEW

**Comment by leexgh** [comment]
there are some my variant info tests failed but I don't think I've touched any of them in this pr, will investigate them

File: service/src/main/java/org/cbioportal/genome_nexus/service/remote/UniprotDataFetcher.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    @Autowired
+    public UniprotDataFetcher(ExternalResourceTransformer<ProteinFeatureInfo> transformer,
+                                       @Value("${uniprot.features.url:https://www.ebi.ac.uk/proteins/api/features/ACCESSION}") String uniprotFeaturesUrl)
+    {
+        super(uniprotFeaturesUrl, MAIN_QUERY_PARAM, PLACEHOLDER);
+        this.transformer = transformer;
+    }
+
+    // TODO: Functions below are not been used for now, will be useful when we have Uniprot data in database
```
[the remark this answers is not in the document]
Comment (leexgh) [reply service/src/main/java/org/cbioportal/genome_nexus/service/remote/UniprotDataFetcher.java:38]: @onursumer Yes I think so, for now we use `fetchInstances` in `BaseExternalResourceFetcher.java`(https://github.com/genome-nexus/genome-nexus/blob/master/service/src/main/java/org/cbioportal/genome_nexus/service/remote/BaseExternalResourceFetcher.java#L60)

**Comment by leexgh** [comment]
@onursumer Yes we don't cache it right now, just fetch data on the fly

**Comment by inodb** [comment]
closing due to inactivity

**Pull request closed without merging**